src/spot.sol

```diff
@@ -84,10 +84,8 @@ contract Spotter is DSNote {
     // --- Update value ---
     function poke(bytes32 ilk) external {
         (bytes32 val, bool zzz) = ilks[ilk].pip.peek();
-        if (zzz) {
-            uint256 spot = rdiv(rdiv(mul(uint(val), 10 ** 9), par), ilks[ilk].mat);
-            vat.file(ilk, "spot", spot);
-            emit Poke(ilk, val, spot);
-        }
+        uint256 spot = zzz ? rdiv(rdiv(mul(uint(val), 10 ** 9), par), ilks[ilk].mat) : 0;
```

Comment:
Why did you remove the `zzz` check? In the case of intentionally setting `spot` to `0` would we not leave the `zzz` to `true` to signal it was different than just an uninitialized value?

Reply:
If the osm is voided: https://github.com/makerdao/osm/blob/master/src/osm.sol#L73 or expired.
Then you will receive `zzz = false`. In that case we would want the `ilk` is "on hold". The way we propose to do that is setting `spot` to zero.

Reply:
That makes sense, I guess I thinking it might be helpful to be able to distinguish between an osm that is voided and one that is just not setup (if I understand this correctly, now they will both return `(0, false)`).  Probably doesn't matter, so lgtm

Reply:
Yeah I guess you will not be able to distinguish that easily, at least if you are not seeing the history of it.